therr-services/maps-service/src/handlers/spaceMetrics.ts

```diff
@@ -70,7 +70,7 @@ const getFormattedMetrics = (startDate, endDate, spaceId) => {
     const currentSeriesPromise = Store.spaceMetrics.getForDateRange(startDate, endDate, { spaceId });
     const prevSeriesPromise = Store.spaceMetrics.getForDateRange(previousSeriesStartDate, (previousSeriesEndDate), { spaceId });
 
-    return Promise.all([currentSeriesPromise, prevSeriesPromise]);
+    return Promise.all([getMetricsByName(currentSeriesPromise, 'space.user.prospect'), getMetricsByName(prevSeriesPromise, 'space.user.prospect'), getMetricsByName(currentSeriesPromise, 'space.user.impression'), getMetricsByName(prevSeriesPromise, 'space.user.impression'), getMetricsByName(currentSeriesPromise, 'space.user.visit'), getMetricsByName(prevSeriesPromise, 'space.user.visit')]);
 };
```

Comment:
Please use the MetricNames constant. We use a lot of enums and constants to prevent typos and keep a single location for renaming or adding to a group of constants